docs/content/docs/build/releases-and-apitags.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Fluid Releases and API Tags
+aliases:
+  - "/build/releases/"
+---
+
+
+## Fluid Release Versions
+
+Fluid Releases will be done through three channels
+
+-   Production
+-   Pre-release
+-   Dev release
+
+### Production Channel
+
+Prod releases are our most stable releases, follow semver and are ready to be used in Production scenarios. These releases correspond to the `latest` tag on npm. This should be the channel used to get the most stable version of Fluid. Following **semver (X.Y.Z)**, this channel will includes:
+
+-   **Major releases (X)** - Used when releasing `breaking changes`. Breaking changes can include API changes, runtime behavior changes and document format changes. Variable release cadence
+-   **Minor releases (Y)** - Used when adding `new features or non-critical bug fixes`, released ~weekly
+-   **Patch releases (Z)** - Used when releasing fixes for `critical, time-sensitive bugs`. Variable release cadence
+
+### Pre-Release Channel (Release Candidates)
+
+Pre-releases are for developers who want to try out early versions of upcoming Fluid releases and provide input as the Major release begins to take shape. `There are no support guarantees for any new or existing APIs and Fluid documents generated by these releases and we recommend not using them in production as they could corrupt pre-existing documents when opened.` However, these are great releases to try out new APIs or test your app with the upcoming release to speed up development. Since pre-releases precede a major release, we can also introduce any breaking changes including API, runtime behavior and document format changes in any RC all the way up to the GA release.
+For example, Fluid containers generated by RC1 are not guaranteed to be compatible with RC3 or the next GA.
+Pre-releases will be represented by the `rc` tags in npm.
+RCs are done with variable timing and go through our full battery of automated tests and always lead up to a major production release.
+We will designate some RCs to be Beta releases when they contain major features that make significant strides towards a Prod release. RCs close to a Prod release will be designated a Preview release.
+There are no guarantees for API breakages or Fluid document compatibility among the release changes. Assume you will have to throw away containers generated by release candidates.
+
+### Dev Release Channel
+
+These are done on demand to preview new APIs or try fixes for our partners. They are represented by the `dev` tag in npm
+There are no guarantees for API breakages or Fluid document compatibility among the release changes. Assume you will have to throw away containers generated by older release candidates
+
+## API Tags
+
+There are no API guarantees for packages in the @fluid-experimental namespace.
```

Comment:
Would this be a good place to also call out that @fluid-internal is not for use by consumers?